src/GraphQL.Relay.Test/GraphQL.Relay.Test.csproj

```diff
@@ -2,10 +2,11 @@
 
   <PropertyGroup>
     <TargetFramework>net5</TargetFramework>
+    <NoWarn>$(NoWarn);1591;IDE1006</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL.SystemTextJson" Version="4.0.2" />
+    <PackageReference Include="GraphQL.SystemTextJson" Version="$(GraphQLTestVersion)" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
     <PackageReference Include="Shouldly" Version="2.8.3" />
```

Comment:
Please bump to 4.0.3

Reply:
Huh? 4.0.3 doesn't exist

Reply:
https://www.nuget.org/packages/Shouldly/4.0.3

Reply:
I only referenced 4.0.2 because we had some critical bugfixes in 4.0.2 and really 4.0.0 and 4.0.1 should not ever be used.  However, this library should work fine with all 4.x versions.

Reply:
OK. I'm talking about Shouldly version.